lib/train/extras/command_wrapper.rb

```diff
@@ -169,8 +169,8 @@ def self.load(transport, options)
         res = LinuxCommand.new(transport, options)
         verification_res = res.verify
         if verification_res
-          msg, reason = verification_res
-          raise Train::UserError, "Sudo failed: #{msg}", reason
+          msg, _reason = verification_res
+          raise Train::UserError, "Sudo failed: #{msg}"
```

Comment:
Wow. Good find... but that `verify` method is just a mess and should be cleaned up (in a different PR). Returning `nil` or a string or an array of a string or a symbol is _not right_. I'll file a separate issue against that. Thank you.

Reply:
@zenspider, will cleanup verify method thanks!